src/components/login/index.module.scss

```diff
@@ -52,6 +52,50 @@
   }
 }
 
+.primaryEmailButton {
+  background-color: var(--color_primary);
+}
+
+.buttonDisabled {
+  background-color: #BDBDBD;
```

Comment:
this hardcode will break the dark mode

Reply:
Reference commit for dark mode if it's helpful: https://github.com/fntechgit/summit-registration-lite/commit/b86859aab3bdfd109996656b7b11f3b1d89b3a26

Reply:
Basically  need to be using css vars for any color so it can be over-ridden on the main site.